ring/basic_lifecycler_test.go

```diff
@@ -102,6 +102,19 @@ func TestBasicLifecycler_RegisterOnStart(t *testing.T) {
 			registerState:  ACTIVE,
 			registerTokens: Tokens{1, 2, 3, 4, 5},
 		},
+		"initial ring contains read only instance": {
+			initialInstanceID: testInstanceID,
+			initialInstanceDesc: &InstanceDesc{
+				Addr:                     "1.1.1.1",
+				State:                    ACTIVE,
+				Tokens:                   Tokens{6, 7, 8, 9, 10},
```

Comment:
I find a bit misleading the different tokens here. I don't think it matters for this specific test. We just want to make sure the instance will still be in RO state, isn't it?

Reply:
Yes that's right. Updated.